src/Microsoft.AspNetCore.SignalR.Common/Protocol/HandshakeProtocol.cs

```diff
@@ -83,6 +99,12 @@ public static void WriteResponseMessage(HandshakeResponseMessage responseMessage
             TextMessageFormatter.WriteRecordSeparator(output);
         }
 
+        /// <summary>
+        /// Creates a new <see cref="HandshakeResponseMessage"/> using the specified bytes.
```

Comment:
Creates a new [HandshakeResponseMessage] **from the specified serialized representation** ~~using the specified bytes~~.